### PR TITLE
Es 1219 cloud run

### PR DESCRIPTION
**Pull request by wrighbr**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -29,14 +65,14 @@ func main() {
...
-		log.Fatal().Err(err).Msg("failed to read github app config")
+		log.Fatal().Err(err).Msg("failed to read github app config:")
 	}
 
-	// create statsd client
-	err = datadog.CreateClient()
-	if err != nil {
-		log.Error().Err(err).Msg("failed to register dogstatsd client")
-	}
+	// // create statsd client
```
Comment (brondum) [inline main.go:71]: Should we remove this, as i guess we dont use it any more ?

File: datadog/client.go [hunk 2]
```diff
@@ -1,47 +1,131 @@
...
-	)
+	metric := metricPrefix + "." + metricName
+	err := postDataDogMetric(metric, int64(value), "gauge", tags)
 	if err != nil {
 		log.Debug().Err(err).Msgf("could send gauge to datadog %s", metricName)
 	}
+
+}
+
+// func getDataDogConfig() (apiKey string, err error) {
```
Comment (brondum) [inline datadog/client.go:49]: Is this needed or should it just be removed ?

Comment (Gaardsholt) [inline datadog/client.go:49]: If this isn't used anymore, then lets delete it :)

File: datadog/client.go [hunk 3]
```diff
@@ -1,47 +1,131 @@
...
+// 	byteValue, _ := ioutil.ReadAll(jsonFile)
+
+// 	var result DatadogConfig
+// 	json.Unmarshal([]byte(byteValue), &result)
+
+// 	return result.Datadog.APIKey, nil
+// }
+
+func postDataDogMetric(metric string, value int64, metricType string, tags []string) error {
+	// apiKey, err := getDataDogConfig()
```
Comment (Gaardsholt) [inline datadog/client.go:67]: See last comment

**Review by Gaardsholt — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (wrighbr) [reply main.go]: removed

File: datadog/client.go (hunk 2 above)
Comment (wrighbr) [reply datadog/client.go]: removed

File: datadog/client.go (hunk 3 above)
Comment (wrighbr) [reply datadog/client.go]: removed

**Pull request merged**